core/internal/settings/settings.go

```diff
@@ -235,6 +240,61 @@ func (s *Settings) GetProgram() string {
 	return s.Proto.Program.GetValue()
 }
 
+// The relative path to the script that created the run, if available.
```

Comment:
is it intended to be relative to anything specific?

Reply:
based on the code seems to be relative to the git root, looks similar to the local_path but relative to the git repo instead of cwd, again some launch craziness, no idea why they need all of these confusing paths.

Reply:
oh wait, no, it could also be relative to the cwd if the root is none, and it seems to be used to construct the config where to look for the "saved" code : https://github.com/wandb/wandb/blob/main/wandb/sdk/wandb_run.py#L700-L701